storage/tests/unit/blob_test.py

```
@@ -1,5 +1,60 @@
+    key, private_key = generate_key(encoding_format)
+
+    signature = Blob.get_pem_signature('test-payload', private_key)
+
+    key.public_key().verify(
+        signature, b'test-payload', padding.PKCS1v15(), hashes.SHA256(),
+    )
+
+
+@pytest.mark.parametrize('private_key', ['', 'not a pem at all'])
+def test_get_pem_signature_invalid_key(private_key):
+    with pytest.raises(ValueError):
```

Ruff (0.16.3)

[warning] `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)